README.org

```diff
@@ -962,6 +963,13 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[https://github.com/jgkamat/alda-mode][alda-mode]] - emacs mode for [[https://github.com/alda-lang/alda][alda]], a music programming language for musicians written in java and clojure.
    - [[https://github.com/supercollider/scel][Scel]] - SuperCollider/Emacs interface
 
+** Screencasting
+
+   - [[https://github.com/tarsius/keycast][keycast]] - Show current commands and its key in the mode line.
+   - [[https://github.com/Ambrevar/emacs-gif-screencast][emacs-gif-screencast]] - Record Emacs frames to gifs.
```

Comment:
Can you update the link to @Ambrevar's gitlab repo: https://gitlab.com/ambrevar/emacs-gif-screencast

Reply:
Oh, yes thanks for catching that!